app/assets/javascripts/index/timeslider.js

```diff
@@ -75,7 +74,7 @@ function addOpenHistoricalMapTimeSlider (map, params, onreadycallback) {
   function getHistoryLayerIfShowing () {
     let ohmlayer;
     map.eachLayer(function (layer) { // there's only 1 or 0 time layers at a time, so this works
-      if (historicalLayerKeys.indexOf(layer.options.layerId) !== -1) ohmlayer = layer;
+      if (layer instanceof L.OSM.OHM) ohmlayer = layer;
```

Comment:
This assumes every instance of `L.OSM.OHM` is time-enabled. That’s currently true, but if we ever choose to publish an all-time cumulative style that isn’t filtered by date, then we’ll need to push this decision down to each instance of `L.OSM.OHM` via a new property in layers.yml.